<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 133, 255);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>11</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 16pt &quot;可口可乐在乎体 楷体&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>关键词提取系统</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>Designed by 谭佐捷，张展鹏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_0">
     <widget class="QTextBrowser" name="textBrowser_result">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>511</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(39, 133, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;选择文件&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="extrapushButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>50</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
    color:White;
    border-style:solid;
    border-radius:60px;
    font-family:微软雅黑;
    background:#2285FF;
}
QPushButton:hover{
    background:#00aaFF;
}
QPushButton:pressed{
    background:#00aaFF;
}</string>
      </property>
      <property name="text">
       <string>提取关键词</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>230</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;提取结果&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>关键词个数</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numlineEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_file">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(34, 133, 255);
color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>清除结果</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(34, 133, 255);</string>
      </property>
      <property name="text">
       <string>BUG反馈</string>
      </property>
     </widget>
     <widget class="QPushButton" name="wordButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(34, 133, 255);</string>
      </property>
      <property name="text">
       <string>生成词云</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="wordimage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>701</width>
        <height>471</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;词云&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="historyButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>历史</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="csvWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>791</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="dehistoryButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>删除记录</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>521</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;使用说明&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;1.点文件选择(三个点)选择要提取的文件。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;2.输入要提取的关键词个数(一般是5)。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;3.点击提取关键词按钮。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;4.如果觉得输出框内容太多，可以点击左侧的清除按钮。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>124</width>
      <height>521</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	font: 12pt &quot;微软雅黑&quot;;
    min-width: 120px;
    max-width: 120px;
    color: Black;
    background: #F5F5F5;
}

QListWidget::Item:selected {
    background: lightGray;
    border-left: 5px solid #2285ff;
}
HistoryPanel:hover {
    background: rgb(52, 52, 52);
}
QListWidget{text-align:right}</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>首页</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>词云</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>历史</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>关于</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>774</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>781</x>
     <y>688</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser_result</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>540</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>extrapushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>keyword()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser_file</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>read()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>-22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wordButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>wordcloud()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>historyButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>history()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>-165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dehistoryButton</sender>
   <signal>clicked()</signal>
   <receiver>csvWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dehistoryButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>dehistory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>-155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>read()</slot>
  <slot>keyword()</slot>
  <slot>wordcloud()</slot>
  <slot>history()</slot>
  <slot>dehistory()</slot>
 </slots>
</ui>
